leetcodePython/Heap/heap_1642.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+from heapq import *
+
+class Solution:
+    def furthestBuilding(self, heights: List[int], bricks: int, ladders: int) -> int:
+        used_ladder_min_heap = []
```

Comment:
Put all ladder first and replace later. The smallest height that use ladder will replace with brick